### PR TITLE
Create Start-SPOTenantRename

### PR DESCRIPTION
**Pull request by WayneEwington**
Documenting the new PowerShell cmdlets to support SPO Tenant Renames.

Please include this PR with the other new files:
* Start-SPOTenantRename
* Get-SPOTenantRenameStatus
* Stop-SPOTenantRename

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+Accept wildcard characters: False
+```
+
+### -WhatIf
+
+Simulation of what would happen if you run the script without modifying anything.
+
+```yaml
+Type: String
+Parameter Sets: UrlAndDestinationDataLocation, UrlAndDestinationUrl
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename:90]: Seems that these parameter sets are not present in the syntax

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+### -DomainName
+
+Specifies the domain name that the current SharePoint domain name will be renamed to. This is the part before "sharepoint.com" or "onmicrosoft.com".
+
+> [!NOTE]
+> The domain name must already have been sucessfully added to Azure AD as per the instructions at [Step 1:Add the new domain name](https://aka.ms/SPOTenantRename#step-1-add-the-new-domain-name). If the domain name does not exist or was not sucessfully added, then this cmdlet will return an error.
+
+```yaml
+Type: String
+Parameter Sets: Default
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename:57]: If there is only one parameter set and it is not named, this probably is just `Parameter Sets: (All)`

**Review by get-itips — COMMENTED** [review]
some small things

**Review by get-itips — COMMENTED** [review]
I just noticed the file you submitted is missing the extension ".md"

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
+
+## RELATED LINKS
+
+[Getting started with SharePoint Online Management Shell](https://docs.microsoft.com/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?view=sharepoint-ps)
+
+[Rename your SharePoint domain](https://aka.ms/SPOTenantRename)
+
+[Get-SPOTenantRenameStatus](Get-TenantRenameStatus.md)
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename:128]:
```suggestion
[Get-SPOTenantRenameStatus](Get-SPOTenantRenameStatus.md)
```

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+[Get-SPOSiteRenameState](Get-TenantSiteState.md)
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename:130]:
```suggestion
[Get-SPOSiteRenameState](Get-SPOSiteRenameState.md)
```

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+
+Prompts you for confirmation before executing the command. For more information, type the following command: `get-help about_commonparameters`
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename:103]:
```suggestion
Prompts you for confirmation before executing the command. For more information, type the following command: `get-help about_commonparameters`.
```

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips and @tseward

**Pull request closed without merging**